cernopendata/modules/fixtures/data/records/atlas-higgs-challenge-2014.json

```diff
@@ -214,17 +214,20 @@
     "accelerator": "CERN-LHC",
     "authors": [
       {
-        "affiliation": "LAL, IN2P3/CNRS & University Paris-Sud, France",
+        "affiliation": "Orsay, LAL; Paris, IN2P3; Orsay",
+        "rorid": "046w3ze59; 03fd77x13; 028rypz17",
         "name": "Adam-Bourdarios, Claire",
         "orcid": "0000-0002-2634-4958"
       },
       {
-        "affiliation": "Physics Department, Royal Holloway, University of London, UK",
+        "affiliation": "Royal Holloway, U. of London",
+        "rorid": "04g2vpn86",
         "name": "Cowan, Glen",
         "orcid": "0000-0001-8363-9827"
       },
       {
-        "affiliation": "TAO team, INRIA & LRI, CNRS & University Paris-Sud, France",
+        "affiliation": "INRIA; LRI, Paris 11; Orsay",
```

Comment:
Can't tell which INRIA it is https://inspirehep.net/institutions?sort=mostrecent&size=25&page=1&q=INRIA

Reply:
If it is Paris-Sud, then it is very probably INRIA Saclay.